### PR TITLE
Add ERC: Authorize Operator

### PR DESCRIPTION
**Pull request by hieronx**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: ERCS/erc-x.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
+---
+eip: x
```
Comment (abcoathup) [inline ERCS/erc-x.md:2]:
```suggestion
eip: 7741
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

File: ERCS/erc-7741.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
+---
+eip: 7741
+title: Authorize Operator
+description: Set Operator via EIP-712 secp256k1 signatures
```
Comment (SamWilsn) [inline ERCS/erc-7741.md:4]: Would be nice to expand on the title a bit more. You have 140-ish characters for a description, so you should use them. Maybe define what an "Operator" is?

File: ERCS/erc-7741.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-06-03
+requires: 712, 1271
+---
+
+## Abstract
+
+A set of functions to enable meta-transactions and atomic interactions with contracts implementing an operator model, via signatures conforming to the [EIP-712](./eip-712.md) typed message signing specification.
```
Comment (SamWilsn) [inline ERCS/erc-7741.md:16]: Again, could probably do with a definition of "operator" here. I assume you mean a role/permission for a contract, and not something like the "unary negation" operator :rofl:

File: ERCS/erc-7741.md [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+
+1. **Enable Meta-Transactions**: Allow users to delegate the execution of transactions to operators, enabling meta-transactions where the user does not need to hold native tokens to pay for gas fees on each chain.
+2. **Improve Security**: Utilize the EIP-712 standard for typed data signing, which provides a more secure and user-friendly way to sign messages compared to raw data signing.
+3. **Facilitate Interoperability**: Provide a standardized interface for operator management that can be adopted across various vault protocols, promoting interoperability and reducing integration complexity for developers.
+4. **Streamline Cross-Chain Operations**: Simplify the process of managing operators across different chains, making it easier for protocols to maintain consistent operator permissions and interactions in a multi-chain environment.
+
+By addressing these needs, the `Authorize Operator` standard aims to streamline the process of managing operators in decentralized vault protocols, making it easier for users and developers to interact with smart contracts in a secure, cost-effective, and interoperable manner across multiple blockchain networks.
```
Comment (SamWilsn) [inline ERCS/erc-7741.md:29]:
```suggestion
By addressing these needs, the Authorize Operator standard aims to streamline the process of managing operators in decentralized vault protocols, making it easier for users and developers to interact with smart contracts in a secure, cost-effective, and interoperable manner across multiple blockchain networks.
```

Only use backticks for inline code.

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**